docs/superpowers/plans/2026-06-27-harry-review-bot.md

```
@@ -0,0 +1,307 @@
+- App 페이지 → Install App → `Moadong/moadong` 에 설치 (Only select repositories)
+
+## 5. 레포 Secret 등록
+`Moadong/moadong` → Settings → Secrets and variables → Actions → New repository secret
+- `CLAUDE_CODE_OAUTH_TOKEN` = 1번 토큰
+- `HARRY_APP_ID` = 3번 App ID
+- `HARRY_APP_PRIVATE_KEY` = 3번 `.pem` 파일 전체 내용 (`-----BEGIN...END-----` 포함)
+
+## 6. 동작 확인
+테스트 PR을 하나 열어 Actions 탭에서 `harry PR Review` 워크플로우 실행 확인.
+PR에 `harry[bot]` 작성자로 코멘트가 달리면 성공.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)